presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -149,6 +155,16 @@ class ResponseHandler : public proxygen::HTTPTransactionHandler {
 
   void onBody(std::unique_ptr<folly::IOBuf> chain) noexcept override {
     if ((chain != nullptr) && !response_->hasError()) {
+      if (type_ == common::PRESTO_ANNOUNCER) {
+        REPORT_ADD_STAT_VALUE(kCounterHttpClientPrestoAnnouncerNumOfOnBody);
+        REPORT_ADD_HISTOGRAM_VALUE(
+            kCounterHttpClientPrestoAnnouncerSizeOfOnBodyChain,
+            chain->length());
```

Comment:
computeChainDataLength() we are not sure if the chain is a single chain.

Reply:
`Announcer` has a hard-coded string which is published to coordinator. It's not changing. Why we need a counter for a message that is constant?